CHANGELOG.md

```diff
@@ -3,6 +3,86 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Mar 3, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📦 Use the same bundling scripts as the core repository (#3965) @flaviendelangle
+
+  The code structure and the bundling strategy has been modified to have a better isolation between components.
+  This brings tree shaking, allowing us to reduce the bundle size. 🏋
+  Such modification are always a bit risky.
+  If you encounter problems with you build, please open an issue.
+  We will put high priority on investigating them.
+
+- 📚 Documentation improvements
+- 🐞 Bug and typo fixes
+
+### `@mui/x-data-grid@v5.6.0` / `@mui/x-data-grid-pro@v5.6.0`
+
+#### Changes
+
+- [DataGrid] Add slot for filter panel delete icon (#4069) @Hameezr
+- [DataGrid] Add specific label for linkOperator (#3915) @alexfauquette
+- [DataGrid] Allow for truncated and multiline content in grid cells (#3955) @DanailH
+- [DataGrid] Allow to navigate between cells with keyboard once inside an `actions` column (#3375) @m4theushw
+- [DataGrid] Fix desynchronization between rows and header when sorting (#4058) @alexfauquette
+- [DataGrid] Clean and document the columns selector (#4010) @flaviendelangle
```

Comment:
Should we highlight the deprecation to provide migration example ?

Reply:
If they were not documented before deprecation I don't think we need to mention.

Reply:
If #4056 is merged the migration will be provided in the warning so we could avoid it. I don't have a strong opinion on this topic. 

For now, migration guides are ready, removing should not take too long

Reply:
So, as I understand the highlight is already present (current starting on line 20), right?
+1 for leaving it.

Reply:
@m4theushw in the future I agree that not documented = no problem
But for the selectors, until a few weeks ago, none were documented but a lot were used in the demos, so I think it's a situation where we should deprecate and communicate about the deprecation.